lighthouse-core/computed/metrics/timing-summary.js

```diff
@@ -43,18 +43,18 @@ class TimingSummary {
     };
 
     const processedTrace = await ProcessedTrace.request(trace, context);
-    const processedNavigation = await ProcessedNavigation.request(processedTrace, context);
+    const processedNavigation = await requestOrUndefined(ProcessedNavigation, processedTrace);
     const speedline = await Speedline.request(trace, context);
-    const firstContentfulPaint = await FirstContentfulPaint.request(metricComputationData, context);
+    const firstContentfulPaint = await requestOrUndefined(FirstContentfulPaint, metricComputationData); // eslint-disable-line max-len
```

Comment:
when a good chunk of code is simple, repetitive and breaks max-len, I like to disable/enable the entire block of code.